ts/features/payments/receipts/screens/ReceiptListScreen.tsx

```diff
@@ -163,14 +165,28 @@ const ReceiptListScreen = () => {
     </>
   );
 
+  type OperationResultEmptyProps = Pick<
+    OperationResultScreenContentProps,
+    "title" | "subtitle" | "pictogram"
+  >;
```

Comment:
Could you declare the type definition outside the component hook?

Reply:
Sure!

Reply:
https://github.com/pagopa/io-app/pull/6575/commits/1d03d3146f8011d5e7f581a1df51d62e04a468e4